src/components/MenuButton.ts

```diff
@@ -10,21 +11,23 @@ const MenuButtonInternal = ({ disabled, children, ...props }: MenuButtonProps, r
   return div({ role: 'menuitem' }, [
     h(
       Clickable,
-      {
-        ref,
-        disabled,
-        style: {
-          display: 'flex',
-          alignItems: 'center',
-          fontSize: 12,
-          minWidth: 125,
-          height: '2.25rem',
-          padding: '0.875rem',
-          ...(disabled ? { color: colors.dark(0.7), cursor: 'not-allowed' } : { cursor: 'pointer' }),
-        } as CSSProperties,
-        hover: !disabled ? { backgroundColor: colors.light(0.4), color: colors.accent() } : undefined,
-        ...props,
-      },
+      _.merge(
```

Comment:
Clarifying question: wasn't the previous code also merging the props using `...props`? 🤔

Reply:
It wasn't merging how I expected. The formatting was off for some reason, adding `_.merge` seemed to have fixed the problem?

Reply:
Ah I see. Thank you for clarifying 👍

Reply:
My guess is because we pass in some extra style props (fontsize) - `_.merge` will combine the styles, whereas simply adding `...props` will overwrite existing styles. Do we know if this affected any other teams/pages that use this component, or are we the only ones passing in additional styles to our usage?